src/MoBi.Core/OSPSuite.Core/Domain/Builder/ExpressionParameter.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OSPSuite.Core.Domain.Builder
+{
+   public class ExpressionParameter : PathAndValueEntity
```

Comment:
I think ParameterStartValue will look the same. MoleculeStartValue has some extra stuff.